examples/Features.ipynb

```diff
@@ -202,7 +202,11 @@
     }
    ],
    "source": [
-    "from altair import Chart, load_dataset\n",
+    "\n",
+    "try: \n",
+    "    from altair import Chart, load_dataset\n",
+    "except TypeError:\n",
```

Comment:
Does altair raise a `TypeError` on older versions? Seems weird. If altair really doesn't work on older Python version, it should raise a deprecation warning or exception. Can you tell what error you encountered?

Reply:
which is what i told them... 
https://github.com/altair-viz/altair/issues/972#issuecomment-418392553
(check the link for the type of error I've gotten)

Apparently Python is partially responsible for the error, so it's complicated...